PayPal.podspec

```
@@ -0,0 +1,27 @@
+Pod::Spec.new do |s|
+  s.name             = "PayPal"
+  s.version          = "0.0.1"
+  s.summary          = "The PayPal iOS SDK: Helps you accept card, PayPal, and alternative payments in your iOS app."
```

nit: It looks like we use "alternative payment methods" instead of "alternative payments" in the docs for BT and PayPal. Should we update this to use the same language?

---

Good call. Let's definitely keep it consistent with the existing docs.
```suggestion
  s.summary          = "The PayPal iOS SDK: Helps you accept card, PayPal, and alternative payment methods in your iOS app."
```